src/main/java/Chat.java

```diff
@@ -0,0 +1,127 @@
+import java.util.*;
+
+public class Chat {
+
+    /**
+     * This Chat version is just supposed to take in the commands,
+     * and return them, thats all. Mitran.Di.CHatri. Ton. Udd. Gayi. Laundi. ah.
+     */
+
+    public static String LINE_BREAK = "---------------";
+    public static String GREETING = "FUNNY FELLA WEIIIIII! \nHow can I help Mr Singhhhhh?";
+    public static String BYE = LINE_BREAK + "\n Byeeeee, come back again ah!\n" +
+            LINE_BREAK;
+    public static String LINE_INTRO = "Nah, here's your list";
+    public static String MARK_MESSAGE = "Power la Mr Bosssssss, mark alr bro!";
+    public static String UNMARK_MESSAGE = "No probs bro, unmarked already!";
```

Comment:
I like that you wanted set this reused phrases as constants! Perhaps consider adding the `final` attribute to them to make it clear that they are constants, and prevent modification?